app/shared/utils/ErrorHandler.scala

```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
-package utils
+package shared.utils
 
-import api.models.errors._
+import _root_.routing.Versions
 import play.api._
 import play.api.http.Status._
 import play.api.mvc.Results._
 import play.api.mvc._
-import _root_.routing.Versions
+import shared.models.errors
+import shared.models.errors._
```

Comment:
See comment about having specific import for `InternalError` 
```suggestion
import shared.models.errors._
```